src/views/Header/HeaderSegment.tsx

```diff
@@ -331,7 +331,7 @@ export default class HeaderSegment extends React.Component<Props, State> {
               style={[
                 Platform.select({
                   ios: null,
-                  default: { left: leftButton ? 72 : 16 },
+                  default: { paddingLeft: leftButton ? 0 : 16 },
```

Comment:
We don't need a padding if there is a `leftButton` as the `title container` will be pushed by the button itself